packages/label/src/Label.js

```diff
@@ -6,9 +6,21 @@ import stylesheet from "./stylesheet";
 
 class Label extends Component {
   static propTypes = {
+    /**
+     * Content of the label, including the label text
+     */
     children: PropTypes.node,
-    disabled: PropTypes.bool, // eslint-disable-line react/no-unused-prop-types
+    /**
+     * Prevents user action pertaining the label
```

Comment:
This comment isn't quite true. User actions aren't effected by this prop, it's purely presentational.

Maybe something like "Dims label text, signifying the related input or control is disabled".

Reply:
OK confirmed that disabling has no effect on events and updated in 5166fd8